### ISSUE TITLE
reCAPTCHA v2 no longer supported

### ISSUE THREAD

**Issue by n2ghtk2n** [alsd4git/chanu#5]
Hi alsd4git,

don't know if you're still interested in updating this app. But it's still the best 4cham viewer ever made.

After recently site updates, we can't post via app anymore, cause reCAPTCHA v2 is sadly no longer supported on 4chan.

Would be really nice if you could check this issue.. If it's not to much work to change.

Thanks mate.

**Comment by BallsDeepWeep** [comment]
Yes, agreed. It would be cool if you left since kind of donation link when you did so we can show some appreciation. I thought I created a request like this quest but I think it must have been on another chanu version within GitHub. This would be a great fix, thanks
